RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZ_vect.cc

```diff
@@ -646,21 +657,43 @@ bool DAClusterizerInZ_vect::purge(vertex_t& y, track_t& tks, double& rho0, const
   double sumpmin = nt;
   unsigned int k0 = nv;
 
+  std::vector<double> inverse_zsums(nt), arg_cache(nt), eik_cache(nt), pmax_cache(nt);
+  double* pinverse_zsums;
+  double* parg_cache;
+  double* peik_cache;
+  double* ppmax_cache;
+  pinverse_zsums = inverse_zsums.data();
+  parg_cache = arg_cache.data();
+  peik_cache = eik_cache.data();
+  ppmax_cache = pmax_cache.data();
```

Comment:
InZT_vect has this times ``` uniquetrkweight_``` defined as ```pcut```.
Please make the implementation as consistent as possible.

Reply:
The original variable name is ```pmax``` and it does not have ```uniquetrkweight``` defined.

Reply:
L689 has ```uniquetrkweight_ * pmax```, which looks the same as pcut from InZT
So, the comment apparently applies

Reply:
OK